lib/xboxrt/libc_extensions/stdlib_ext_.h

```diff
@@ -17,6 +17,8 @@ typedef void (__cdecl *_purecall_handler)(void);
 _purecall_handler __cdecl _get_purecall_handler (void);
 _purecall_handler __cdecl _set_purecall_handler (_purecall_handler function);
 
+#define _sys_nerr _PDCLIB_ERRNO_MAX
```

Comment:
`#include "pdclib/_PDCLIB_config.h"` is only ever done indirectly in the pdclib stdlib.h
Should we do it explicitly here, or just assume that is has happened?

Reply:
I think it's fine, afaik the config is (indirectly) included in every header.